src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Unix.cs

```diff
@@ -62,6 +62,7 @@ public static partial class PlatformDetection
 
         public static bool IsDrawingSupported { get; } = GetGdiplusIsAvailable();
         public static bool IsSoundPlaySupported { get; } = false;
+        public static bool SupportsSsl3 => (PlatformDetection.IsOSX || (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) && PlatformDetection.OpenSslVersion < new Version(1, 0, 2) && !PlatformDetection.IsDebian));
```

Comment:
What happened to the `!PlatformDetection.IsRedHatFamily6` part?

Reply:
There were two places when this was detected. One had IsRedHatFamily6 and one did not. I'll watch it as it seems like we do not run RH6 on PRs. (and build on that platform is real pain)

Reply:
@rmkerr added that check in https://github.com/dotnet/corefx/pull/28998; presumably it was due to failures in official runs?

Reply:
I did more testing on RH6 repro machine using openssl s_client/s_server and curl and I can negotiate sslv2:

```
[dotnet-bot@a0009HS ~]$ curl -sslv2 -v -k https://127.0.0.1:4433/
*   Trying 127.0.0.1...
* Connected to 127.0.0.1 (127.0.0.1) port 4433 (#0)
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/pki/tls/certs/ca-bundle.crt
  CApath: none
* SSLv2 (OUT), , Client hello (1):
* SSLv2 (IN), , Server hello (4):
* SSLv2 (OUT), , Client key (2):
* SSLv2 (OUT), , Client finished (3):
* SSLv2 (IN), , Server verify (5):
* SSLv2 (IN), , Server finished (6):
* SSL connection using SSLv2 / DES-CBC3-MD5
```

Reply:
RH6 failed _only_ for platform handler. 
And I posted wrong snipped yesterday.

```
[dotnet-bot@a0009HS ~]$ curl -v -ssl3 -k https://127.0.0.1:4433/
*   Trying 127.0.0.1...
* Connected to 127.0.0.1 (127.0.0.1) port 4433 (#0)
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/pki/tls/certs/ca-bundle.crt
  CApath: none
* SSLv3 (OUT), TLS handshake, Client hello (1):
* SSLv3 (IN), TLS handshake, Server hello (2):
* SSLv3 (IN), TLS handshake, Certificate (11):
* SSLv3 (IN), TLS handshake, Server key exchange (12):
* SSLv3 (IN), TLS handshake, Server finished (14):
* SSLv3 (OUT), TLS handshake, Client key exchange (16):
* SSLv3 (OUT), TLS change cipher, Client hello (1):
* SSLv3 (OUT), TLS handshake, Finished (20):
* SSLv3 (IN), TLS change cipher, Client hello (1):
* SSLv3 (IN), TLS handshake, Finished (20):
```

Since the platform is capable of SSLv3 I'll fix up the specific test instead of changing SupportsSsl3.